src/script/team/TeamEntity.ts

```diff
@@ -30,7 +30,7 @@ export class TeamEntity {
   /** Team icon (asset key) */
   iconKey?: string;
   id?: string;
-  members: ko.ObservableArray<TeamMemberEntity>;
+  members: ko.ObservableArray<User>;
```

Comment:
Are you sure about that? There is a difference between "member" and "user" properties.

Reply:
Yes, the types were wrong.

Reply:
I re-checked and we get a `User` back :)